spec/integration/check_clean_slate_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'lite_spec_helper'
+
+# This test can be used to manually verify that there are no leaked
+# background threads - execute it after executing another test (in the same
+# rspec run) that is suspected to leak background threads.
```

Comment:
Could you provide some guidance for how to run this test in the same rspec run as another test?

Reply:
It's literally just `rspec foo_spec.rb spec/integration/check_clean_slate_spec.rb`.  How would you like me to phrase that?

Reply:
Oh that's great. I didn't know you could do that. Could you just include that in the comment?
```
execute it after executing another test in the same rspec run with the following command:

rspec your_spec.rb spec/integration/check_clean_slate_spec.rb
```

Does that sound good?

Reply:
Yup, added to the comment thanks.

Reply:
Thanks!